sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -210,26 +207,25 @@ export interface NamespaceProperties {
     modifiedAt: Date;
     name: string;
     namespaceType: string;
-}
+
```

Comment:
Looks like these formatting changes shouldn't have checked in.
Please revert.

Reply:
The commit is reverted.